### PR TITLE
✨ description added using CURL

### LINKED ISSUE #8179: [Docs] Add example to use authenticated REST endpoints with curl
#### Current State
The current [REST API docs](https://docs.meshery.io/reference/rest-apis) do not contain an example on how to access authenticated endpoints using the `curl` command.

#### Desired State
Add an example of how to use a token in a `curl` command to access Meshery server's authenticated endpoints.

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by spacemc2**
**Notes for Reviewers**

This PR fixes #8179 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@spacemc2 excellent!

File: docs/pages/reference/rest-api-reference.md [hunk 1]
```diff
@@ -56,12 +57,32 @@ Using <a href='/reference/mesheryctl'>Meshery CLI</a>, you can get a copy of you
...
+
+<details>
+  <summary>Example using curl</summary>
+  <br />
+Using curl, you can access Meshery's REST API by executing this command:
+  <br/>
+  <br/>
+  <pre class="codeblock-pre">
+  <div class="codeblock"><div class="clipboardjs">curl --location 'localhost:9081/api/&lt;endpoint&gt;' \
+--header 'meshery-token: &lt;mytoken&gt;\
```
Comment (asubedy) [inline docs/pages/reference/rest-api-reference.md:74]:
```suggestion
--header 'meshery-token: &lt;yourToken&gt;\
```

File: docs/pages/reference/rest-api-reference.md [hunk 2]
```diff
@@ -56,12 +57,32 @@ Using <a href='/reference/mesheryctl'>Meshery CLI</a>, you can get a copy of you
...
+--header 'Cookie: meshery-provider=Meshery; meshery.layer5.io_ref=/;token=&lt;mytoken&gt;
```
Comment (asubedy) [inline docs/pages/reference/rest-api-reference.md:75]:
```suggestion
--header 'Cookie: meshery-provider=Meshery; meshery.layer5.io_ref=/;token=&lt;yourToken&gt;
```

**Review by asubedy — CHANGES_REQUESTED** [review]

**Review by spacemc2 — COMMENTED** [review]
The suggestions were resolved!

**Comment by l5io** [comment]
🚀 Preview for commit 016d3a9fff7e18b8d0d8e855a3f1bbb6b3630ef8 at: https://64becfed90cf16116b8ab377--meshery-docs-preview.netlify.app

**Comment by Chadha93** [comment]
@spacemc2  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Review by leecalcote — APPROVED** [review]
Most excellent, @spacemc2 👏

**Pull request merged**